runtime/doc/options.txt

```diff
@@ -5147,6 +5144,12 @@ A jump table for the options with a short description can be found at |Q_op|.
 
 	This option cannot be set from a |modeline| or in the |sandbox|, for
 	security reasons.
+							*shell-powershell*
+	To use powershell (on Windows): >
+		set shell=powershell shellquote=\" shellpipe=\| shellredir=>
```

Comment:
`set shellquote=(` is more robust because of current shell quoting rules.
`call termopen(['powershell', '-NoProfile', '-Command', 'echo a > foo.txt', '>', '$null'])` results in a redirection error. Wrapping the `echo` command with double quotes results in the same error. Workaround is to wrap the command `(` and `)`.

`powershell -NoProfile -Command "echo a > foo.txt" > $null` should work though and it works with `term_start` on Vim.

Reply:
This help snippet was copied from the test helpers.lua. should we change it?

Reply:
Sure but the builds should be unaffected.

Reply:
@janlazo will `set shellquote=(` be future-compatible with your PR that changes the quoting behavior?

Reply:
Yes. I'm considering `set shellquote=(` for cmd.exe as well.

On Neovim, I explicitly unset `set shellxquote` on my vimrc but my PR shouldn't affect it.
On Vim, `set shellxquote=\"` is required for cmd.exe to pass the entire command to powershell.